Swap.cpp

```diff
@@ -0,0 +1,55 @@
+#include<iostream>
+using namespace std;
+
+void SwapRef(int &a, int &b) 
+{
+   int temp;
+   temp = a; 
+   a = b;    
+   b = temp; 
+     
+   return;
+}
+
+void SwapPoint(int *a, int *b)
+{
+
+   int temp;
+   temp = *a;   
+   *a = *b;     
+   *b = temp;   
+  
+   return;
+}
+
+int Func (int A[],int n)
+{
+	int i;
+	for (i=0;i<n;i++)
+	 cout<<*(A+i)<<" ";
+	
+}
+
+
+int main ()
```

Comment:
Your swap functions are both correct. Good job on that. However you probably didnt understand my question well so I'll explain again. 

Question : receive an array as input and ask user for two indexes . for the user entered indexes , swap those two elements in array using one of the three function : 

function 1 : swap using call by reference 
function 2 : swap using pointers 
function 3 : pass the entire array as function parameter along with the indexes to swap from , and then swap. 

Input & Output : 

Enter the size of array : 5
Enter the elements : 10 20 30 40 50
Enter the two indexes you wish to swap : 0 3
Choose a method (1,2,3) to swap : 2 
Array before swapping : Printing array : 10 20 30 40 50
Array after swapping : Printing array : 40 20 30 10 50
Code completed. Exiting .... 


You have done well implementing function 1 , and function 2. implement function 3 and change the main code to fit the code question.